src/callbacks.py

```diff
@@ -208,9 +239,11 @@ def update_geofigure(selected_year, selected_type, selected_value):
             tooltip=[
                 alt.Tooltip("country:N", title="Country"),
                 alt.Tooltip(
-                    f"{y_column}:Q",
-                    title=f"{'Absolute' if selected_type == 'absolute' else 'Relative'} {'Incidence' if selected_value == 'incidence' else 'Mortality'}",
-                ),  # Corrected dynamic variable reference and format
+                    field=y_column,
+                    type="quantitative",
+                    title=f"{'Relative' if selected_type == 'relative' else 'Absolute'} {'Incidence' if selected_value == 'incidence' else 'Mortality'}",
+                    format=".1%" if selected_type == "relative" else "",
+                ),
```

Comment:
Looks good to me.
But since here seems to be the only place that was changed.
I would suggest only changing this part, not formatting the other part of the code.

One way to make it is to only edit this part in a pure text editor other than IDE with linter (like VScode); Or edit it in github online interface.